src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx

```diff
@@ -0,0 +1,224 @@
+      }
+    }
+
+    const podValue = 0;
+    const pod = this.props.pods[podValue];
+    const containerInfo = this.getContainerInfo(pod);
+
+    this.state = {
+      containerInfo: containerInfo,
+      duration: DurationDefault,
+      loadingPodLogs: false,
+      podValue: podValue
+    };
+  }
+
+  componentDidMount() {
+    if (this.state.containerInfo) {
+      const pod = this.props.pods[this.state.podValue!];
+      this.fetchLogs(this.props.namespace, pod.name, this.state.containerInfo.container, Number(this.state.duration));
+    }
+  }
+
+  componentDidUpdate(prevProps: WorkloadPodLogsProps, prevState: WorkloadPodLogsState) {
+    const prevContainer = prevState.containerInfo ? prevState.containerInfo.container : undefined;
+    const newContainer = this.state.containerInfo ? this.state.containerInfo.container : undefined;
+    const updateContainer = newContainer && newContainer !== prevContainer;
+    const updateDuration = this.state.duration && prevState.duration !== this.state.duration;
+    if (updateContainer || updateDuration) {
+      const pod = this.props.pods[this.state.podValue!];
+      this.fetchLogs(this.props.namespace, pod.name, newContainer!, Number(this.state.duration));
+    }
+  }
+
+  render() {
+    return (
+      <>
+        {this.state.containerInfo && (
+          <>
+            <Toolbar>
+              <ToolbarDropdown
+                id={'wpl_pods'}
+                nameDropdown="Pod"
+                tooltip="Display logs for the selected pod"
+                handleSelect={key => this.setPod(key)}
+                value={this.state.podValue}
+                label={this.props.pods[this.state.podValue!].name}
+                options={this.podOptions!}
+              />
+              <ToolbarDropdown
+                id={'wpl_containers'}
+                nameDropdown="&nbsp;&nbsp;&nbsp;Container"
+                tooltip="Display logs for the selected pod container"
+                handleSelect={key => this.setContainer(key)}
+                value={this.state.containerInfo.container}
+                label={this.state.containerInfo.container}
+                options={this.state.containerInfo.containerOptions!}
+              />
+              <Toolbar.RightContent>
+                <ToolbarDropdown
+                  id={'wpl_duration'}
+                  handleSelect={key => this.setDuration(key)}
+                  value={this.state.duration}
+                  label={DurationOptions[this.state.duration]}
+                  options={DurationOptions}
+                  tooltip={'Time range for graph data'}
+                />
+                <span style={{ paddingLeft: '0.5em' }}>
+                  <Button id={'wpl_refresh'} disabled={!this.state.podLogs} onClick={() => this.handleRefresh()}>
+                    <Icon name="refresh" />
+                  </Button>
+                </span>
+              </Toolbar.RightContent>
+            </Toolbar>
+            <textarea
```

Comment:
Please move all of the inline styles to type-style.

Reply:
Ok, I’ll move to type style